### PR TITLE
feat: added product-listing-page

### PR DESCRIPTION
**Pull request by Radheshyam1234**

### CODE REVIEW

File: product-listing-page.html [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+            <span class="badge-container">
+              <i class="fas fa-shopping-cart">
+                <span class="badge-icon badge-pink">1</span></i
+              >
+            </span>
+          </a>
+        </div>
+      </header>
+
+      <sidebar class="sidebar">
```
Comment (lannisterrr) [inline product-listing-page.html]: is "sidebar" a semantic UI tag? If not then use "aside" for sidebar.

File: product-listing-page.html [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+          <label>
+            <input type="radio" name="sort-price" /> Price - Low to High
+          </label>
+          <label>
+            <input type="radio" name="sort-price" checked /> Price - High to Low
+          </label>
+        </div>
+      </sidebar>
+
+      <main>
```
Comment (lannisterrr) [inline product-listing-page.html]: You can also "section" tag .

**Review by lannisterrr — COMMENTED** [review]
Other than points mentioned above , the code seems fine.

File: product-listing-page.html (hunk 1 above)
Comment (Radheshyam1234) [reply product-listing-page.html]: Done

File: product-listing-page.html (hunk 2 above)
Comment (Radheshyam1234) [reply product-listing-page.html]: Done

File: product-listing-page.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.filter {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+}
+.price-filter .price-range {
+  justify-content: space-between;
+  margin-bottom: 10px;
```
Comment (ramakrishnach14) [inline product-listing-page.css:32]: better to use relative units

**Review by ramakrishnach14 — COMMENTED** [review]
Looks Good , in spacing either margin or padding better to maintain uniformity.

**Pull request merged**